pkg/controllers/nodeclaim/lifecycle/initialization_test.go

```diff
@@ -366,6 +366,18 @@ var _ = Describe("Initialization", func() {
 		Expect(ExpectStatusConditionExists(nodeClaim, v1.ConditionTypeInitialized).Status).To(Equal(metav1.ConditionTrue))
 	})
 	It("should not consider the Node to be initialized when all startupTaints aren't removed", func() {
+		startupTaints := []corev1.Taint{
```

Comment:
Curious, why do we need to make these test changes?

Reply:
I think was part of cleanup that I was doing earlier when this was not using labels as the feature flag. I can revert if you would like.

Reply:
Yeah, let's drop this change if it's not needed

Reply:
Sure